<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>777</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../build/Server/resources.qrc">:/resources/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>61</pointsize>
           </font>
          </property>
          <property name="text">
           <string>塔台监控系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>断开</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>服务器状态：</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="3" column="1">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>2h</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>用户登陆数量：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>持续运行时间：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>服务器IP：</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>192.168.1.100</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>正常运行</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>运行状态：</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Log：</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QTextEdit" name="textEdit">
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;09:18:19&lt;/span&gt;:启动服务器&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;09:20:19&lt;/span&gt;:配置用户&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;09:22:19&lt;/span&gt;:配置设备&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;09:25:19&lt;/span&gt;:为用户配置了对空相机-设备1&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;09:30:19&lt;/span&gt;:服务器开始监听&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;09:35:19&lt;/span&gt;:admin已连接&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>用户登陆状态：</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QTableWidget" name="tableWidget">
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="showGrid">
                <bool>true</bool>
               </property>
               <property name="sortingEnabled">
                <bool>false</bool>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="cornerButtonEnabled">
                <bool>true</bool>
               </property>
               <row>
                <property name="text">
                 <string>1</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>2</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>3</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>用户</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>用户权限</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>登陆状态</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>对空相机</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>对地相机</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>直显相机</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string>admin</string>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string>admin</string>
                </property>
               </item>
               <item row="0" column="2">
                <property name="text">
                 <string>OK</string>
                </property>
               </item>
               <item row="0" column="3">
                <property name="text">
                 <string>0</string>
                </property>
               </item>
               <item row="0" column="4">
                <property name="text">
                 <string>0</string>
                </property>
               </item>
               <item row="0" column="5">
                <property name="text">
                 <string>2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_config">
    <property name="title">
     <string>服务器配置</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_userconfig"/>
    <addaction name="action_devconfig"/>
    <addaction name="action_nvrconfig"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu_config"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_userconfig">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="action_devconfig">
   <property name="text">
    <string>设备管理</string>
   </property>
  </action>
  <action name="action_nvrconfig">
   <property name="text">
    <string>NVR配置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../build/Server/resources.qrc"/>
 </resources>
 <connections/>
</ui>
